packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -306,6 +339,27 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
         CurvedAnimation(parent: entranceController, curve: Curves.easeIn);
     allEntrance =
         CurvedAnimation(parent: entranceController, curve: Curves.easeIn);
+    changeController = AnimationController(
+      vsync: this,
+      duration: entranceAnimationDuration,
+    )
+      ..addStatusListener((status) {
+        if (status == AnimationStatus.completed) {
+          setState(() {
+            _properties = _animatedProperties.end;
+            _animatedProperties = null;
+            changeController.value = 0.0;
+          });
+        }
+      })
+      ..addListener(() {
+        print(
+            'Changing ${changeController.value}: ${_animatedProperties?.children?.elementAt(1)?.size}');
+      });
```

Comment:
do we want to keep the print statement? if yes then it's fine to ignore previous nit